lib/main_common.pm

```diff
@@ -829,20 +845,6 @@ sub load_inst_tests {
         if (get_var("SYSTEM_ROLE_FIRST_FLOW")) {
             load_system_role_tests;
         }
-        #system_role selection during installation was added as a new feature since sles12sp2
-        #so system_role.pm should be loaded for all tests that actually install to versions over sles12sp2
-        #no matter with or without INSTALL_TO_OTHERS tag
-        if (
-            is_sle
-            && (check_var('ARCH', 'x86_64')
-                && sle_version_at_least('12-SP2')
-                && is_server()
-                && (!is_sles4sap() || is_sles4sap_standard())
-                && (install_this_version() || install_to_other_at_least('12-SP2'))
-                || is_sle('15+')))
-        {
-            loadtest "installation/system_role";
```

Comment:
We still need to schedule this module here

Reply:
that code I merged in second and third condition in line 551. It looks like for sle12sp2 there is some specifics plus sle15+

Reply:
I see your point, but because it's spaghetti code, we won't get same schedule if `SYSTEM_ROLE_FIRST_FLOW` is not set. Therefore to play it safe I would not aim multiple changes at the same time. @mnowaksuse WDYT?

Reply:
Should not affect JeOS.

Reply:
Sorry, got confused by collapsed lines. By still seems wrong to me, as SYSTEM_ROLE_FIRST_FLOW is set only for TW installations. So maybe then we should trigger load_system_role_tests unconditionally.

Reply:
I will send another PR, I think for sle will not pass the condition, you are right.